### DISCUSSION TITLE
Hydrological Simulation on a small scale simplified test model

### DISCUSSION THREAD

**Discussion by lulukef** [mhm-ufz/mHM#44] [Ideas]
Dear mHM team,

I am trying to use mHM to simulate the hydrological process for rainfall infiltration into the slope and couple it with OGS (OpenGeoSys) to simulate the subsurface flow governed by the Richards equation and failure process. As far as I Know, mHM by its name "the mesoscale Hydrological Model" is designed to simulate the hydrological process to a very large horizontal extent. 

Is there anyone trying or having experience in preparing mHM input files for the arbitrarily created model like below? I am just curious if it is possible we could check how accurate mHM can simulate the process in a small scale model (like in [this paper](https://www.sciencedirect.com/science/article/abs/pii/S1364815221000025)).
![image](https://user-images.githubusercontent.com/55043585/140925757-71d63766-7ad0-4f9f-a923-f7386a50e49e.png)

Regards,
Laxman Kafle

**Comment by StephanThober** [comment]
Dear @laxman360 ,
to the best of my knowledge, we do not have specific tools to create synthetic input data (if I understand you correctly). Please let me know if this discussion is still relevant to you.
Best,
Stephan

**Reply by lulukef** [reply]
Dear Dr. Stephan Thober,

Thank you very much for your reply. I have stopped working on mHM#OGS6 coupling just for now and as far as I know one team from UFZ currently working on this to develop a coupling tool. 
For the synthetic input data preparation, that was my initial plan to use mHM#OGS6 to check applicability in a small-scale model. Since the coupling is currently not possible between mHM-OGS6, I had abandoned this project. It will be interesting to simulate this problem after mHM#OGS6 coupling code becomes available. 

Regards,
Laxman Kafle
